academic-server/academic-server-rest/src/main/java/by/kovzov/uis/academic/rest/controller/CurriculumController.java

```diff
@@ -28,9 +31,24 @@ public Page<CurriculumDto> getAll(@PageableDefault(sort = "approvalDate", direct
         return curriculumService.getAll(pageable);
     }
 
+    @GetMapping
+    @PreAuthorize("hasAuthority('CURRICULUM_GET')")
+    public Page<CurriculumDto> search(@RequestBody SearchDto searchDto,
```

Comment:
If I remember correctly `@RequestBody` doesn't work with `@GetMapping`.

Reply:
I am not sure either but I don't want to think about this now